requirements.txt

```diff
@@ -1,8 +1,7 @@
 coloredlogs
-Cython>=0.28.4
 inicheck>=0.9.0,<0.10.0
 netCDF4>=1.2.9
-numpy>=1.14.0,<1.19.0
+numpy>=1.14.0
```

Comment:
Curious if a bigger than a specific version needs to be there? My understanding is that it will always take the latest and 1.14 was released in January of 2018. I would feel pretty confident at this point that it won't get anything older than that.

Reply:
I did not manually copy, exported the data frame to csv then compared. So sorta manual.

Probably could do away with the minimum bound now.